src/RpcClient/TransactionManager.cs

```diff
@@ -104,7 +104,7 @@ private long EstimateNetworkFee()
             foreach (var hash in hashes)
             {
                 size += 166;
-                networkFee += ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES64] + ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES33] + InteropService.GetPrice(InteropService.Neo_Crypto_ECDsaVerify, null);
+                networkFee += ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES64] + ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES33] + InteropService.GetPrice(InteropService.Crypto.ECDsaVerify, null);
             }
```

Comment:
Also this opcodes need to be changed

Reply:
Please check it, i am not sure if it's well estimated

Reply:
I think it is good, @shargon, and well detailed!

Reply:
This test is failing still: https://github.com/neo-project/neo-modules/blob/3fddad5e9b8d60211c635afe80d8c29176a582fb/tests/Neo.Network.RPC.Tests/UT_TransactionManager.cs#L124

Reply:
Skip(2)